src/components/activityEventElems/ActivityElement/activityElementEvent.ts

```diff
@@ -0,0 +1,5 @@
+import { PayEvent } from 'models/subgraph-entities/vX/pay-event'
+
+export type ActivityElementEvent = Pick<PayEvent, 'timestamp' | 'txHash'> & {
```

Comment:
if this is a generic activity type, maybe we shouldn't tie it to a specific event (PayEvent)? and just define the properties explicitly